pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -2867,6 +2867,27 @@ func TestChangefeedErrors(t *testing.T) {
 		t, `unknown on_error: not_valid, valid values are 'pause' and 'fail'`,
 		`CREATE CHANGEFEED FOR foo into $1 WITH on_error='not_valid'`,
 		`kafka://nope`)
+
```

Comment:
Let's move this test into a separate test fixture.  (This particular test deals with error)